Build seems to be broken

I cloned the repo using 

git clone https://github.com/googlemaps/google-maps-services-js.git

and when I ran

npm install

I received several errors when the typescript compiler ran:

src/adapter.ts:59:9 - error TS2322: Type 'any' is not assignable to type 'never'.

59         r.data = transformData(r.data, r.headers, config.transformResponse);
           ~~~~~~

src/adapter.ts:60:40 - error TS2339: Property 'status' does not exist on type 'never'.

60         if (r.status === 200 && r.data.status) {
                                          ~~~~~~

src/adapter.ts:61:42 - error TS2339: Property 'status' does not exist on type 'never'.

61           r.status = statusToCode(r.data.status);
                                            ~~~~~~

src/directions.ts:185:18 - error TS2430: Interface 'DirectionsResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'DirectionsResponseData' is not assignable to type 'never'.

185 export interface DirectionsResponse extends AxiosResponse {
                     ~~~~~~~~~~~~~~~~~~

src/distance.ts:165:18 - error TS2430: Interface 'DistanceMatrixResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'DistanceMatrixResponseData' is not assignable to type 'never'.

165 export interface DistanceMatrixResponse extends AxiosResponse {
                     ~~~~~~~~~~~~~~~~~~~~~~

src/elevation.ts:68:18 - error TS2430: Interface 'ElevationResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'ElevationResponseData' is not assignable to type 'never'.

68 export interface ElevationResponse extends AxiosResponse {
                    ~~~~~~~~~~~~~~~~~

src/geocode/geocode.ts:96:18 - error TS2430: Interface 'GeocodeResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'GeocodeResponseData' is not assignable to type 'never'.

96 export interface GeocodeResponse extends AxiosResponse {
                    ~~~~~~~~~~~~~~~

src/geocode/reversegeocode.ts:96:18 - error TS2430: Interface 'ReverseGeocodeResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'ReverseGeocodeResponseData' is not assignable to type 'never'.

96 export interface ReverseGeocodeResponse extends AxiosResponse {
                    ~~~~~~~~~~~~~~~~~~~~~~

src/geolocate.ts:58:18 - error TS2430: Interface 'GeolocateResponseSuccess' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'GeolocateResponseData' is not assignable to type 'never'.

58 export interface GeolocateResponseSuccess extends AxiosResponse {
                    ~~~~~~~~~~~~~~~~~~~~~~~~

src/geolocate.ts:66:18 - error TS2430: Interface 'GeolocateResponseError' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type '{ error: { code: number; message: string; errors: { domain: string; reason: GeolocateErrorReason; message: string; }[]; }; }' is not assignable to type 'never'.

66 export interface GeolocateResponseError extends AxiosResponse {
                    ~~~~~~~~~~~~~~~~~~~~~~

src/places/autocomplete.ts:167:18 - error TS2430: Interface 'PlaceAutocompleteResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'PlaceAutocompleteResponseData' is not assignable to type 'never'.

167 export interface PlaceAutocompleteResponse extends AxiosResponse {
                     ~~~~~~~~~~~~~~~~~~~~~~~~~

src/places/details.ts:63:18 - error TS2430: Interface 'PlaceDetailsResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'PlaceDetailsResponseData' is not assignable to type 'never'.

63 export interface PlaceDetailsResponse extends AxiosResponse {
                    ~~~~~~~~~~~~~~~~~~~~

src/places/findplacefromtext.ts:51:18 - error TS2430: Interface 'FindPlaceFromTextResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'FindPlaceFromTextResponseData' is not assignable to type 'never'.

51 export interface FindPlaceFromTextResponse extends AxiosResponse {
                    ~~~~~~~~~~~~~~~~~~~~~~~~~

src/places/placesnearby.ts:111:18 - error TS2430: Interface 'PlacesNearbyResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'PlacesNearbyResponseData' is not assignable to type 'never'.

111 export interface PlacesNearbyResponse extends AxiosResponse {
                     ~~~~~~~~~~~~~~~~~~~~

src/places/queryautocomplete.ts:89:18 - error TS2430: Interface 'PlaceQueryAutocompleteResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'PlaceQueryAutocompleteResponseData' is not assignable to type 'never'.

89 export interface PlaceQueryAutocompleteResponse extends AxiosResponse {
                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/places/textsearch.ts:94:18 - error TS2430: Interface 'TextSearchResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'TextSearchResponseData' is not assignable to type 'never'.

94 export interface TextSearchResponse extends AxiosResponse {
                    ~~~~~~~~~~~~~~~~~~

src/roads/nearestroads.ts:33:18 - error TS2430: Interface 'NearestRoadsResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type '{ snappedPoints: SnappedPoint[]; }' is not assignable to type 'never'.

33 export interface NearestRoadsResponse extends AxiosResponse {
                    ~~~~~~~~~~~~~~~~~~~~

src/roads/snaptoroads.ts:48:18 - error TS2430: Interface 'SnapToRoadsResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type '{ snappedPoints: SnappedPoint[]; }' is not assignable to type 'never'.

48 export interface SnapToRoadsResponse extends AxiosResponse {
                    ~~~~~~~~~~~~~~~~~~~

src/timezone.ts:69:18 - error TS2430: Interface 'TimeZoneResponse' incorrectly extends interface 'AxiosResponse<never>'.
  Types of property 'data' are incompatible.
    Type 'TimeZoneResponseData' is not assignable to type 'never'.

69 export interface TimeZoneResponse extends AxiosResponse {
                    ~~~~~~~~~~~~~~~~


Found 19 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @googlemaps/google-maps-services-js@3.2.5 prepare: `tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the @googlemaps/google-maps-services-js@3.2.5 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.


I am guessing an update to Axios did this.

--- a/package.json
+++ b/package.json
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "agentkeepalive": "^4.1.0",
-    "axios": "^0.22.0",
+    "axios": "^0.21.1",
     "crypto-js": "4.1.1",
     "query-string": "^7.0.1",
     "retry-axios": "^2.2.1"

